From 248e5891b1623a4beba143a05588c641992ca00a Mon Sep 17 00:00:00 2001
From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Sat, 12 Aug 2017 22:52:14 -0400
Subject: [PATCH 2/3] gbp.conf: Drop configuration options that should be local

There's no need to try to push people to use pbuilder
---
 debian/gbp.conf | 2 --
 1 file changed, 2 deletions(-)

diff --git a/debian/gbp.conf b/debian/gbp.conf
index e6efcb2d..c181fae8 100644
--- a/debian/gbp.conf
+++ b/debian/gbp.conf
@@ -11,5 +11,3 @@ color = auto
 upstream-tree = tag
 # uncomment this to automatically GPG sign tags:
 sign-tags = True
-postbuild = lintian $GBP_CHANGES_FILE
-pbuilder = True
-- 
2.11.0

